### DISCUSSION TITLE
웹 접근성과 색의 관계

### DISCUSSION THREAD

**Discussion by hozzijeong** [woowacourse-teams/2023-pium#170] [질문있어요] [answered]
웹 접근성을 고려해서 코드를 작성하고 있는데 색의 변동이 어쩔수 없이 발생하게 되었습니다. 

---

저희가 사용하는 PrimaryColor와 backgroundColor의 색 비율이 2.12:1이 나옵니다 (작은 텍스트 기준 4.5:1이 나와야 합니다) 이 부분을 어떻게 해야할까요...? 글자 색 비율이란게 폰트 크기에 따라서 그 비율이 달라지긴 하는데, 전부 고정해서 맞추는게 나을지 아니면 일단 너무 눈에 띄는 색은 수정하되 나머지는 기존의 방식대로 갈지 여러분의 생각을 듣고싶습니다.

<img src="https://github.com/woowacourse-teams/2023-pium/assets/50974359/278e7f9f-788f-4067-bce0-7f251b69e61d" width="200px" />

<img src="https://github.com/woowacourse-teams/2023-pium/assets/50974359/e89a4b3b-9bd7-42f2-9d12-432a0ba4a2a3" width="200px"/>

**Comment by Choi-JJunho** [comment]
웹 접근성에 맞춰서 수정한 결과가 아래 사진이라고 인지하면 될까요?
<img src="https://github.com/woowacourse-teams/2023-pium/assets/50974359/e89a4b3b-9bd7-42f2-9d12-432a0ba4a2a3" width="400px"/>
저는 웹 접근성을 고려한 색 비율을 따라가도 괜찮아보입니다~

**Reply by WaiNaat** [reply]
요게 바뀐 결과라면 저도 찬성합니다
색 비율 기준이 생각보다 세네요ㅋㅋ

**Reply by hozzijeong** [reply]
약간 골때리는게... 폰트 크기에 따라서 글자가 바뀌는 겁니다... 이거 기준 잡기가 상당히 어렵네요 ㅠㅠ

**Reply by WaiNaat** [reply, marked as answer]
깔끔하게 444444로 통일은 어떠신가요

**Reply by Choi-JJunho** [reply]
저도 둘중 하나가 좋다면 444444에 한표입니다

**Reply by hozzijeong** [reply]
ㅋㅋㅋㅋㅋㅋ 확인했습니다!
